content/en/security/application_security/guide/manage_account_theft_appsec.md

```diff
@@ -0,0 +1,669 @@
+
+After setting up instrumentation for your services, ASM monitord for attack campaigns. You can review the monitoring in the [Attacks overview][14] **Business logic** section. 
+
+<!-- ![][image10] -->
+
+ASM detects [multiple attacker strategies][15]. Upon detecting an attack with a high level of confidence, the [built-in detection rules][16] generate a signal. 
+
+The severity of the signal is set based on the urgency of the threat: from **Low** in case of unsuccessful attacks to **Critical** in case of successful account compromises.
+
+To fully leverage detections, take the following actions.
+
+### Step 2.1: Configuring notifications
+
+[Notifications][17] provide warnings when a signal is triggered. 
+
+To create a notification rule using the [Create a new rule][18] setting, do the following:
+
+1. Open [Create a new rule][18].  
+2. Enter a name for the rule.
+3. Select **Signal** and remove all entries except **ASM**.
+4. Restrict the rule to `category:account_takeover.`  
+5. Add notification recipients (Slack, Teams, PagerDuty).
+   To learn more, see [Notification channels][19].  
+6. Save the rule.
+   <!-- ![][image11] -->
+   The notification is sent the next time a signal is generated.
+
+### Step 2.2: Validate proper data propagation
+
+In microservice environments, services are generally reached by internal hosts running other services. This internal environment makes it challenging to identify the unique traits of the attacker, such as IP, user agent, fingerprint, etc., and validate the data.
+
+[ASM Traces][20] can help to validate the data by exposing the source IPs and user agent traffic.
+
+To validate the data, do the following:
+
+1. Review login traces in the [Traces][21] and check for the following:  
+* Source IPs (`@http.client_ip`) are varied and public IPs.  
+  * **Problem:** If login attempts are coming from a few IPs only, this might be  a proxy that you can't block without risking availability.  
+  * **Solution:** Forward the client IP of the initial request through a HTTP header, such as `X-Forwarded-For`. You can use a custom header for [better security][22] and configure the tracer to read it using the `DD_TRACE_CLIENT_IP_HEADER` environment variable.  
+* The user agent (`@http.user_agent`) is consistent with the expected traffic (web browser, mobile app, etc.)  
+  * **Problem:** The user agent could be replaced by the user agent in the calling microservice network library.  
+  * **Solution:** Use the client user agent when calling subsequent services.
+
+### Step 2.3: Configure automatic blocking
+
+**Before you begin:** Verify that the IP addresses are properly configured, as described in [Step 2.2: Validate proper data propagation](#step-2.2:-validate-proper-data-propagation).
+
+ASM automatic blocking can be used to block attacks at any time of the day. Automatic blocking can help block attacks before your team members are online, providing security during off hours.
```

Comment:
```suggestion
ASM automatic blocking can be used to block attacks at any time of the day. Automatic blocking can help block attacks before your team members are online, providing security during off hours. Within the context of an ATO, it can help mitigate load issues caused by the increase in failed login attempts or prevent compromised accounts from being used by the attacker.
```